src/System.Linq/samples/project.json

```diff
@@ -0,0 +1,13 @@
+{
+  "dependencies": {
+    "System.Collections": "4.0.10",
```

Comment:
Apologies if any of this misses the point you're making:

If I don't include `System.Collections` as a dependency it will fail to compile due to not having `Equals`, `GetHashCode`, or `get_Default`.  I can get the dependency list down to `System.Collections`, `System.Runtime`, and `System.Console` before it gives me any warnings or errors.

This project doesn't contain any dependencies that the test project doesn't already have (including the `System.Console 4.0.0-beta-*` dependency).  Any dependency issues with this project would also exist in the test project. We don't plan for any sample project to have any more dependencies than a test project in the same solution -- if any changes break a test project, they should also break a samples project (and vice/versa).